oe_corporate_blocks.post_update.php

```diff
@@ -130,13 +136,23 @@ function oe_corporate_blocks_post_update_30001(): void {
  * Import updated EC and EU footer data, along with their translations.
  */
 function oe_corporate_blocks_post_update_30002(): void {
-  _oe_corporate_blocks_import_corporate_links();
+  $config_path = drupal_get_path('module', 'oe_corporate_blocks') . '/config/post_update/30002_update_footer_data';
+  _oe_corporate_blocks_import_corporate_links($config_path);
 }
 
 /**
- * Create default footer link sections.
+ * Introduce new entity type and create default footer link sections.
  */
 function oe_corporate_blocks_post_update_30003(): void {
+  // Install new 'Footer Link Section' config entity type.
+  $update_manager = \Drupal::entityDefinitionUpdateManager();
```

Comment:
From what I see in Dany's comment in slack this should be done in hook_update_N

Reply:
I've pushed update.